libs/user-identity-service/src/main/java/uk/gov/di/ipv/core/library/service/UserIdentityService.java

```diff
@@ -191,10 +190,12 @@ public boolean isVcSuccessful(List<VcStatusDto> currentVcStatuses, String criIss
 
     public boolean areVcsCorrelated(List<VerifiableCredential> vcs)
             throws HttpResponseExceptionWithErrorBody, CredentialParseException {
-        var successfulVcs =
-                VcHelper.filterVCBasedOnProfileType(vcs, ProfileType.GPG45).stream()
-                        .filter(VcHelper::isSuccessfulVc)
-                        .toList();
+        var successfulVcs = new ArrayList<VerifiableCredential>();
```

Comment:
Another case of not being able to use streams due do `isSuccessfulVc` throwing a checked exception.